AlphaWallet/Foundation/Repeat.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Repeat.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 14/3/22.
+//
+
+import Foundation
+
+func repeatTimes(_ times: Int, block: () -> Void) {
```

Comment:
I'm skeptical about adding this. The `for-each` look is perfectly fine. I'd much rather not create new dialects. For some exceptions, it's useful to create some utility functions; in this case, the old version requires an explicit `0` and choice of inclusive/exclusive-ended loops, so maybe it's useful. So let's keep it for now.

Reply:
I'll take this out.